src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/Models/Manifest.cs

```diff
@@ -103,4 +106,69 @@ public class Blob
         [XmlAttribute(AttributeName = "NonShipping")]
         public bool NonShipping { get; set; }
     }
+
+    [XmlRoot(ElementName = "SigningInformation")]
+    public class SigningInformation
+    {
+        [XmlAttribute(AttributeName = "AzureDevOpsCollectionUri")]
```

Comment:
I don't have a lot of context in this signing work, but why are we duplicating the azdo build information in this section?

Reply:
@jonfortescue, do you know? I think what @riarenas means is that we already have build info in the `Build` node

Reply:
Yeah I don't think this would be needed?

Reply:
This comes from [here](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L252). Tbh I'm not entirely sure what it does; I based my models on @chcosta's signing manifest which was designed to be used in the release pipeline. I'm not entirely sure off the top of my head what the intent was.